docs/02-import-models.rst

```diff
@@ -1,12 +1,11 @@
-
 Import models
 =============
 
 Import models from files
 ------------------------
 
-The function `~cameo.io.load_model` accepts a number of different
-input formats.
+The function :class:`~cameo.io.load_model` accepts a number of
```

Comment:
I guess this would better be [:py:func:](https://www.sphinx-doc.org/en/master/usage/restructuredtext/domains.html#cross-referencing-python-objects).

```suggestion
The function :py:func:`~cameo.io.load_model` accepts a number of
```